components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -81,6 +81,18 @@ Refer to the following pages on how to create users and assign roles in ServiceN
 - [Creating Users in ServiceNow](https://docs.servicenow.com/csh?topicname=t_CreateAUser.html)
 - [Assigning Roles in ServiceNow](https://docs.servicenow.com/csh?topicname=t_AssignARoleToAUser.html)
 
+### Discovery Source
+
+The Chef Automate Integration App provides a unique ID as a discovery source that distinguishes between discovered Configuration Items (CIs) and imported CIs. The **Discovery Source** field is part of the `cmdb_servers` table, and it takes **ChefAutomate** value while importing CIs by default.
+
+{{< figure src="/images/automate/snow_integration_discovery.png" alt="Discovery Source">}}
+
+{{< note >}}
+
+The servers recognized by Chef Automate Integration App have discovery source value as **ChefAutomate**.
+
+{{< /note >}}
```

Comment:
We can lose this note since the content is covered in the paragraph before the picture.
```suggestion
```

Reply:
I guess its required.. as those servers recognized by the app will be taking up ChefAutomate value. In note, again we are emphasizing them. Isnt it?

Reply:
I don't think we need it because it's almost exactly the same text from the line before the image. It's just redundant. Notes are best used sparingly and only when necessary.